src/system/connectivity/blynk.cpp

```diff
@@ -0,0 +1,160 @@
+#include "blynk.h"
+
+#if ARDUINO_ARCH_ESP32
+#include <WiFi.h>
+#include <BlynkSimpleEsp32.h>
+#include <WiFiClient.h>
+#elif ARDUINO_ARCH_ESP8266
+#include <ESP8266WiFi.h>
+#include <BlynkSimpleEsp8266.h>
+#else
+#error "Unsupported ARDUINO architecture"
+#endif
+
+#include "utils/mem.h"
+#include "utils/logger.h"
+#include "utils/storage.h"
+
+static dev_conf_t temp_inf;
+static uint8_t temp_relay;
+
+void KlicBlynk::init(const char *_auth)
+{
+  Blynk.config(_auth);
+}
+
+void KlicBlynk::run()
+{
+  Blynk.run();
+}
+
+void KlicBlynk::showStatus(int r,bool s)
+{
+  switch(r)
+  {
+    case 0: //turn on relay 1
+      Blynk.virtualWrite(V7,s);
+      Blynk.virtualWrite(V8,0);
+      Blynk.virtualWrite(V9,0);
+      break;
+    case 1:
+      Blynk.virtualWrite(V7,0);
+      Blynk.virtualWrite(V8,s);
+      Blynk.virtualWrite(V9,0);
+      break;
+    case 2:
+      Blynk.virtualWrite(V7,s);
+      Blynk.virtualWrite(V8,s);
+      Blynk.virtualWrite(V9,0);
+      break;
+    case 3:
+      Blynk.virtualWrite(V7,0);
+      Blynk.virtualWrite(V8,0);
+      Blynk.virtualWrite(V9,s);
+      break;
+    default:
+      Blynk.virtualWrite(V7,0);
+      Blynk.virtualWrite(V8,0);
+      Blynk.virtualWrite(V9,0);
+  }
+}
+
+dev_conf_t getConfFromMem(uint8_t _relay)
+{
+  dev_conf_t t;
+  eeprom_map_conf_time_t r;
+  ZeroMem(&t, sizeof(dev_conf_t));
+  r = storage.getConfiguration(_relay);
+  t.hora = r.hour;
+  t.min = r.min;
+  t.duration = (r.duration / 60); // duration saved in seconds and show in min in Blynk
```

Comment:
Change the 60 for a macro like:
```suggestion
  t.duration = (r.duration / SECONDS_IN_A_MINUTE); // duration saved in seconds and show in min in Blynk
```